README.md

```diff
@@ -82,8 +82,12 @@ Returns a new focus trap on `element` (one or more "containers" of tabbable node
 
 #### createOptions
 
-- **onActivate** `{() => void}`: A function that will be called when the focus trap activates.
-- **onDeactivate** `{() => void}`: A function that will be called when the focus trap deactivates,
+- **onActivate** `{() => void}`: A function that will be called **before** sending focus to the target element upon activation.
+- **onPostActivate** `{() => void}`: A function that will be called **after** sending focus to the target element upon activation.
+- **checkCanFocusTrap** `{(containers: Array<HTMLElement | SVGElement | string | () => HTMLElement | SVGElement>) => Promise<unknown>}`: Animated dialogs have a small delay between when `onActivate` is called and when the focus trap is focusable. `checkCanFocusTrap` expects a promise to be returned. When that promise resolves, focus will be sent to the focus-trap. Due to the lack of Promise support, `checkCanFocusTrap` is not supported in IE11 unless you provide a Promise polyfill.
+- **onDeactivate** `{() => void}`: A function that will be called **before** sending focus to the trigger element upon deactivation.
```

Comment:
This relates to the `returnFocus` / `returnFocusOnDeactivate` option. I think we should change the description here to say, "A function that will be called **before** returning focus to the node focused prior to activation (or configured with the `setReturnFocus` option) upon deactivation."